cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -48,14 +48,18 @@ message SourceSegyFile {
     string cloud_storage_path = 7; // The cloud storage path for the file, excluding the file name
     map<string, string> metadata = 5;
     SegyOverrides segy_overrides = 6;
+    repeated TraceHeaderField trace_header_fields = 8; // The trace header fields that will be used as keys for indexing.
 }
 
 // Metadata related to interpreting SEG-Y files.
 message SegyOverrides {
-    google.protobuf.Int32Value inline_offset = 1; // Inline number field in the trace headers. Defaults to 189 as per the SEG-Y rev1 specification
-    google.protobuf.Int32Value crossline_offset = 2; // Crossline number field in the trace headers. Defaults to 193 as per the SEG-Y rev1 specification
-    google.protobuf.Int32Value cdp_x_offset = 3; // X coordinate of ensemble (CDP) position in trace headers. Defaults to 181 as per the SEG-Y rev1 specification
-    google.protobuf.Int32Value cdp_y_offset = 4; // Y coordinate of ensemble (CDP) position in trace headers. Defaults to 185 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value inline_offset = 1;                  // Inline number field in the trace headers. Defaults to 189 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value crossline_offset = 2;               // Crossline number field in the trace headers. Defaults to 193 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value cdp_x_offset = 3;                   // X coordinate of ensemble (CDP) position in trace headers. Defaults to 181 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value cdp_y_offset = 4;                   // Y coordinate of ensemble (CDP) position in trace headers. Defaults to 185 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value cdp_offset = 6;                     // Ensemble (CDP) position in trace headers. Defaults to 21 as per the SEG-Y rev1 specification
```

Comment:
Seeing it together with `cdp_x` and `cdp_y` makes me do a double-take. Maybe it should be `cdp_number_offset`?

Genuinely unsure. Maybe it's fine.

Reply:
I did rename it.. May be clearer